src/routes/safe/components/Transactions/TxsTable/columns.js

```diff
@@ -34,29 +33,40 @@ type TxData = {
 
 export const formatDate = (date: string): string => format(parseISO(date), 'MMM d, yyyy - HH:mm:ss')
 
-export const getIncomingTxAmount = (tx: IncomingTransaction) => {
-  const txAmount = tx.value ? `${new BigNumber(tx.value).div(`1e${tx.decimals}`).toFixed()}` : 'n/a'
-  return `${txAmount} ${tx.symbol || 'n/a'}`
+type TxValues = {
+  value?: string | number,
+  decimals?: string | number,
+  symbol?: string,
 }
 
-export const getTxAmount = (tx: Transaction) => {
-  const web3 = getWeb3()
-  const { fromWei, toBN } = web3.utils
-
-  let txAmount = 'n/a'
-
-  if (tx.isTokenTransfer && tx.decodedParams) {
-    const tokenDecimals = tx.decimals.toNumber ? tx.decimals.toNumber() : tx.decimals
-    txAmount = `${formatAmount(
-      BigNumber(tx.decodedParams.value)
-        .div(10 ** tokenDecimals)
-        .toString(),
-    )} ${tx.symbol}`
-  } else if (Number(tx.value) > 0) {
-    txAmount = `${fromWei(toBN(tx.value), 'ether')} ${tx.symbol}`
+const NOT_AVAILABLE = 'n/a'
+
+const getAmountWithSymbol = ({ decimals = 0, symbol = NOT_AVAILABLE, value }: TxValues, usePrecise = false) => {
+  const nonFormattedValue = BigNumber(value).times(`1e-${decimals}`).toFixed()
+  const finalValue = usePrecise ? formatAmount(nonFormattedValue).toString() : nonFormattedValue
```

Comment:
Sorry I think you got me wrong, my point was not that something wrong with the name, but with the usage. Precise = non formatted output, used where every number matters.
Here:
```
usePrecise ? formatAmount(nonFormattedValue).toString() : nonFormattedValue
```
if precise = true is passed, the formatted amount is returned, but should be vice versa

To keep the usage the same, you may use `formatted` as a name. I don't really like the idea of naming thing with use, I want to keep this for hooks

Reply:
Ok, let's go with `formatted` then

Reply:
Formatted https://github.com/gnosis/safe-react/pull/812/commits/00a8c02433986604c3155ece86a8f9656ef6b4de